app/src/integrationTest/kotlin/maru/app/MaruFollowerTest.kt

```diff
@@ -247,4 +323,18 @@ class MaruFollowerTest {
         assertThat(blocksImportedByFollower).isEqualTo(blocksProducedByQbftValidator)
       }
   }
+
+  private fun checkStackBlocks(
```

Comment:
I didn't catch from the name that you meant "network stack". Can you please add it to the name? Or better, keep the name simple `checkBlocksOn` and rename `stack` to `networkStack` and `blocksToProduce` to `blocksExpected`